### PR TITLE
chmod: add a+rX example

### PR DESCRIPTION
**Pull request by blaylockbk**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples. **This is the 8th example**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

This new example for chmod is one I often need to look up when someone asks "hey, can you make it so I can see the contents of that directory?" The capital `X` is the secret sauce to this command because it makes subdirectories executable but not files. That way the other user can navigate all the subdirectories. 

The use of capital X in chmod is also explained on this page: https://prasadlinuxblog.wordpress.com/2012/05/09/the-uppercase-x-in-chmod/

Thanks for considering this new example.

### CODE REVIEW

File: pages/common/chmod.md [hunk 1]
```diff
@@ -30,3 +30,7 @@
 - Change permissions recursively giving [g]roup and [o]thers the ability to [w]rite:
 
 `chmod -R g+w,o+w {{directory}}`
+
+- Recursively give [a]ll users [r]ead permissions to all contents in a directory. Makes subdirectories e[X]ecutable but not files:
```
Comment (marchersimon) [inline pages/common/chmod.md:34]: What do you think about this?
```suggestion
- Recursively give [a]ll users [r]ead and e[X]ecute permissions to a directory and its sub-directories, but not to the files it contains:
```

Comment (blaylockbk) [reply pages/common/chmod.md:34]: It's better, but makes it sound like that only directories get read access, when files also get read access. Is this more clear?
```suggestion
- Recursively give [a]ll users [r]ead permissions to files and e[X]ecute permissions to sub-directories within a directory:
```

Comment (marchersimon) [reply pages/common/chmod.md:34]: Perfect, I like it.

File: pages/common/chmod.md [hunk 2]
```diff
@@ -30,3 +30,7 @@
 - Change permissions recursively giving [g]roup and [o]thers the ability to [w]rite:
 
 `chmod -R g+w,o+w {{directory}}`
+
+- Recursively give [a]ll users [r]ead permissions to files and e[X]ecute permissions to sub-directories within a directory:
+
+`chmod -R a+rX {{directory}}`
```
Comment (marchersimon) [inline pages/common/chmod.md:36]:
```suggestion
`chmod -R a+rX {{directory}}`

```
This should fix the error from `tldr-bot`.

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @blaylockbk

**Comment by blaylockbk** [comment]
You're welcome! And thanks for providing an opportunity to learn how to contribute to an open-source project. Even though this was a small addition, the experience was valuable.

**Comment by marchersimon** [comment]
Feel free to contribute something any time you want. We always need volunteers to make tldr better.

**Pull request merged**